types/src/traits/signature_key/bn254/bn254_priv.rs

```diff
@@ -0,0 +1,71 @@
+use custom_debug::Debug;
+use jf_primitives::signatures::bls_over_bn254::{KeyPair as QCKeyPair, SignKey as QCSignKey};
+use rand::SeedableRng;
+use rand_chacha::ChaCha20Rng;
+use serde::{Deserialize, Serialize};
+use std::cmp::Ordering;
+
+/// Private key type for a bn254 keypair
+#[derive(PartialEq, Eq, Clone, Serialize, Deserialize, Debug)]
+pub struct BN254Priv {
```

Comment:
As mentioned above, the naming for this wrapping should mention the signature scheme instead of the curve.